### PR TITLE
link to Nexus 2 documentation

### PR DESCRIPTION
**Pull request by hboutemy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,15 +18,16 @@ Collection of Apache Maven plugins supporting Nexus Suite.
 
 ## Plugins available
 
-* [Nexus Staging Plugin](https://github.com/sonatype/nexus-maven-plugins/tree/master/staging/maven-plugin) - Maven Plugin to perform Sonatype Nexus Staging Workflow steps from your build. 
-* [Nexus M2Settings Maven Plugin](https://github.com/sonatype/nexus-maven-plugins/tree/master/m2settings/maven-plugin) - Maven Plugin to download a settings.xml file templated stored in Nexus and potentially replace user token values.
+* [Nexus Staging Plugin](https://github.com/sonatype/nexus-maven-plugins/tree/master/staging/maven-plugin) - Maven Plugin to perform [Sonatype Nexus Staging](https://help.sonatype.com/repomanager2/staging-releases/staging-overview) workflow steps from your build. 
+* [Nexus M2Settings Maven Plugin](https://github.com/sonatype/nexus-maven-plugins/tree/master/m2settings/maven-plugin) - Maven Plugin to [manage Maven settings](https://help.sonatype.com/repomanager2/managing-maven-settings), ie. download a settings.xml file templated stored in Nexus and potentially replace user token values.
```
Comment (lisadurant) [inline README.md:22]: * [Nexus M2Settings Maven Plugin](https://github.com/sonatype/nexus-maven-plugins/tree/master/m2settings/maven-plugin) - Maven Plugin to [manage Maven settings](https://help.sonatype.com/repomanager2/managing-maven-settings) (i.e., download a settings.xml file template stored in Nexus and potentially replace user token values).

File: staging/maven-plugin/README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@
 -->
 # Nexus Staging Maven Plugin
 
-Maven Plugin to control Nexus Staging workflow. While the maven plugin ("staging client") part is OSS, to use staging features it need a Sonatype Nexus Professional instance 2.1+ on the server side!
+Maven Plugin to control [Nexus Staging](https://help.sonatype.com/repomanager2/staging-releases/staging-overview) workflow. While the maven plugin ("staging client") part is OSS, to use staging features it need a Sonatype Nexus Professional instance 2.x on the server side!
```
Comment (lisadurant) [inline staging/maven-plugin/README.md:17]: Maven Plugin to control [Nexus Staging](https://help.sonatype.com/repomanager2/staging-releases/staging-overview) workflow. While the Maven Plugin ("staging client") part is OSS, you must have a Sonatype Nexus Professional 2.x instance on the server side to use staging features.

File: README.md (hunk 1 above)
Comment (hboutemy) [reply README.md]: @lisadurant reworded as proposed in [117ccb0](https://github.com/sonatype/nexus-maven-plugins/pull/94/commits/117ccb0f0167a5c34e0c4452b8cc3518004f884b)

File: staging/maven-plugin/README.md (hunk 2 above)
Comment (hboutemy) [reply staging/maven-plugin/README.md]: @lisadurant reworded as proposed in [117ccb0](https://github.com/sonatype/nexus-maven-plugins/pull/94/commits/117ccb0f0167a5c34e0c4452b8cc3518004f884b)

**Comment by hboutemy** [comment]
thank you @lisadurant for review and proposed improvements

**Pull request merged**